### PR TITLE
Add spec URLs for RTCPeerConnection to RTCRtpSender

### PR DESCRIPTION
**Pull request by sideshowbarker**

### CODE REVIEW

File: api/RTCPeerConnection.json [hunk 1]
```diff
@@ -1394,6 +1395,7 @@
       "getLocalStreams": {
         "__compat": {
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/RTCPeerConnection/getLocalStreams",
+          "spec_url": "https://w3c.github.io/webrtc-pc/#dom-peerconnection-getsenders",
```
Comment (Elchi3) [inline api/RTCPeerConnection.json:1398]: This looks wrong

File: api/RTCPeerConnection.json [hunk 2]
```diff
@@ -1506,6 +1508,7 @@
       "getRemoteStreams": {
         "__compat": {
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/RTCPeerConnection/getRemoteStreams",
+          "spec_url": "https://w3c.github.io/webrtc-pc/#dom-peerconnection-getreceivers",
```
Comment (Elchi3) [inline api/RTCPeerConnection.json:1511]: This isn't matching

**Review by Elchi3 — CHANGES_REQUESTED** [review]

File: api/RTCPeerConnection.json (hunk 1 above)
Comment (sideshowbarker) [reply api/RTCPeerConnection.json]: `getLocalStreams` is deprecated and standard_track:false, and not a member of RTCPeerConnection in the spec, and https://developer.mozilla.org/docs/Web/API/RTCPeerConnection/getLocalStreams redirects to https://developer.mozilla.org/en-US/docs/Web/API/RTCPeerConnection/getSenders — so I’ve updated the patch to drop the `spec_url` here.

File: api/RTCPeerConnection.json (hunk 2 above)
Comment (sideshowbarker) [reply api/RTCPeerConnection.json]: `getRemoteStreams` is deprecated and standard_track:false, and not a member of RTCPeerConnection in the spec, and https://developer.mozilla.org/docs/Web/API/RTCPeerConnection/getRemoteStreams redirects to https://developer.mozilla.org/en-US/docs/Web/API/RTCPeerConnection/getReceivers, so I’ve updated the patch here to drop the `spec_url` here.

**Review by Elchi3 — APPROVED** [review]

**Pull request merged**